rfc/0016-specification-repository.md

```diff
@@ -0,0 +1,54 @@
+This is an issue when a major part of that RFC needs to be redefined--the entire RFC template is excluded for these changes! This is fine for clarifications, but in large-scale changes that require lots of discussion, the current process works against the point of defining these documents through RFCs in the first place.
+
+
+# Explanation
+
+The `quiltmc/specifications` repository created by this RFC can hold three major types of specifications:
+
+1. File specifications
+2. Governance policy
+3. Technical definitions (see: ASMR, QSL)
+   For simplicity, each of these kinds of documents will be called specifications within this RFC.
+   Each specification would be its own file.
+   Creation or major revisions (to be determined by a case-by-case basis) of a specification requires an RFC.
+   Clarifications and other minor revisions can be made by making a PR directly to the specification repository.
+
+Each specification only contains the actionable parts of the RFCs involved (usually summary and explaination). Motivation, Rationale, Unresolved Questions, etc. are left only on the RFC.
+
+## Requirements for Creation
+
+- Creation of a specification is required for file specifications and governance policy.
+
+- Creation of a specification for technical definitions is at the author's discretion, but may be required in cases where major parts of the document are expected to be revised, or if an amendment PR to the RFC repository is deemed as "too big" and needing its own RFC.
+
+## Other details
+How general policy is split into specifications is not meant to be 1 rfc -> 1 file; maintainers should go for clarity and ease of reference, with refactors where needed.
+
+# Drawbacks
+
+- This change, if accepted, increases the complexity of submitting an RFC to Quilt, by requiring an additional PR to be made after acceptance of most RFCs. Additionally, it is not clear when an RFC will be required for a revision, which might discourage people from contributing revisions to Quilt specifications.
+- This change assumes that it is not desired for RFCs to be the central authority on policy, but instead to provide a look at the initial motivation and intentions behind a change *before* it is implemented.
+
+# Unresolved Questions
+
+- Should the restrictions on technical definitions be stricter? relaxed? or are they fine?
+- Is this change even needed, or will the current amendment process be fine?
+
+# Rationale and Alternatives
+
+- Having a specifications repository keeps all official documents about policy in one spot.
+- We could continue with the existing process, but having policy mixed between RFCs is not desirable as described in the Motivation section above.
+
+
+# Expected Response
+
+This change should make it easier for the community to locate a central authority on policy and specifications, and encourage long-form discussion of major revisions to community policy through the proper RFC process.
```

Comment:
> a central authority on policy and specifications

This is what the RFC repository is for. Adding a specification repository would make it confusing because you might not be sure where something is specified. Quilt projects, wiki pages, website pages, etc could all just reference an RFC when they need to.

Reply:
How would you handle situations where an RFC is replaced by a new one?

Reply:
The new RFC would mention something like "This standard supersedes the one specified in RFC 123" and a small edit is made to the old RFC like "This RFC is obsoleted by RFC 234". As prior art, here's the first example I came across: https://datatracker.ietf.org/doc/html/rfc2822.